### PR TITLE
enhance and add several Amber-related functions

### PR DESCRIPTION
**Pull request by njzjz**
<del>* move all constants into one file</del>
* enhance amber functions
* add several functions

### CODE REVIEW

**Comment by njzjz** [comment]
It seems that there are many versions of unit converting constant...

**Comment by njzjz** [comment]
@amcadmus

File: tests/test_cp2k_aimd_output.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def setUp (self) :
         self.system_1 = dpdata.LabeledSystem('cp2k/aimd',fmt='cp2k/aimd_output')
         self.system_2 = dpdata.LabeledSystem('cp2k/aimd/deepmd', fmt='deepmd/raw')
         self.places = 6
-        self.e_places = 6
+        self.e_places = 1
```
Comment (amcadmus) [inline tests/test_cp2k_aimd_output.py:12]: Why do you change the place of precision?

**Comment by amcadmus** [comment]
I do not see UT for the `amber` module nor the new methods added to the `LabeledSystem`. Please add.

**Comment by njzjz** [comment]
I am planning to split this PR into two PRs.

**Comment by njzjz** [comment]
> I do not see UT for the `amber` module nor the new methods added to the `LabeledSystem`. Please add.

ready to review

**Comment by amcadmus** [comment]
Could you please add UTs for `pick_atom_idx`, `remove_atom_names` and `pick_by_amber_mask`? 

I understand that they are tested in `TestAmberCorr`, but the test is not so "unit". Once any bug happens in these methods, it would be not easy to locate them.

**Comment by njzjz** [comment]
I have split the unittests.

**Pull request merged**